tree/30_generic_methods/kernel_module_enabled_at_boot.cf

```diff
@@ -0,0 +1,78 @@
+#######################################################################################
+# Copyright 2019 Normation SAS
+#####################################################################################
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, Version 3.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+#####################################################################################
+
+# @name Kernel module enabled at boot
+# @description Ensure that a given kernel module is enabled on the system
```

Comment:
wait, is it enabled now, or enabled at boot only, or both ?

Reply:
enabled at boot, enabled does not make sense for a module.